test/model/mapping/coercer_test.rb

```diff
@@ -16,22 +17,41 @@
       coercer.to_record(entity).must_equal(name: 'Tyrion Lannister')
     end
 
+    describe 'new record' do
+      # Bug:
+      # https://github.com/lotus/model/issues/155
+      it 'ignores unset values' do
+        entity = User.new(name: 'Daenerys Targaryen')
+        coercer.to_record(entity).must_equal(name: 'Daenerys Targaryen')
+      end
+
+      it 'forces nil values' do
```

Comment:
this describe block is when you create new record, we test two cases, new entity without force an attribute to nil and another case forcing a nil attribute.

`to_record` method translate the entity to a format compatible with the database, then nil attributes shouldn't be sent when are creating a entity, but in this case you don't persirst an entity, only we are testing `to_record` method getting the correct values